config/v1/types_apiserver.go

```diff
@@ -30,6 +30,15 @@ type APIServerSpec struct {
 	// - ConfigMap.Data["ca-bundle.crt"] - CA bundle.
 	// +optional
 	ClientCA ConfigMapNameReference `json:"clientCA"`
+	// extraCORSAllowedOrigins is a field of allowed CORS that should be added on top of the basic CORS needed by the
+	// infrastructure in order to allow in-cluster components to access the API and OAuth servers using Javascript.
+	// The field values should follow the syntax of the corsAllowedOrigins field.
```

Comment:
Be specific here: note that each item is a regex and include an example.

Reply:
Suggestion, short and crisp: "extraCORSAllowedOrigins lists additional, user-defined domains for which the API server allows access from different domains using the CORS headers. This is needed to access the API and OAuth from JavaScript applications."